### ISSUE TITLE
Feature Request: Add external iCalendar support

### ISSUE THREAD

**Issue by cniesen** [etesync/etesync-web#58]
Often a person has multiple calendars (e.g private etesync calendar and office360 work calendar) that they may need to see in a single view.  This feature request would allow the user to add an iCalendar to their secure etesyc-web calendar without having to compromise the security of their etesync calendar.

Possible user scenario:

* User selects "Add iCalendar" icon/link.
* Add iCalendar modal displays the following fields:
  * Calendar Title [text field]
  * URL [text field for ics url]
  * Color [shows list of colors]
  * Store in browser (don't enable this on shared computers) [check box to enable HTML5 Web Storage of url]
  * OK/Cancel buttons
* iCalendar events are displayed in line with etesync events (different colors)
* Calendar title is displayed on the side with a show/hide toggle button/link/icon and a remove button/link/icon

**Comment by tasn** [comment]
I think this is better than #18 indeed. This would never allow editing/creation though, just access publicly available resources. Implementing full DAV is definitely out of scope.

**Comment by tasn** [comment]
Sure, but implementing this one is trivial, the other one is so much work that it's likely it would never be done. :)

**Comment by betsythefc** [comment]
I currently use 3 different calendar apps for syncing:
Etesync, DAV5x and ICS5x. It would be really cool if part of etesync let me sync normal DAV and ICS calendars, and synced those URLs between the android and web client, so I only need the one app, and the one web client.

**Comment by tasn** [comment]
Yeah, I agree it would be nice. Though DAV is immensely complex, so not sure it's worth it... ICS is much easier and I think DAV servers expose the data in a way that's similar to ICS (exactly the same?) so we can probably at least have one-way (read-only) DAV feeds, which should be enough for many use-cases.

In summary: not sure full-blown DAV is worth it.

**Comment by betsythefc** [comment]
For my use case, read only ICS/DAV is just fine.

Thanks,
Bryce McNab
Sent from ProtonMail mobile

-------- Original Message --------

**Comment by tasn** [comment]
I bet it's the same for most people too...

**Comment by andreasmjg** [comment]
Hi there, I'd like read-only ICS-pulling feature as well.

**Comment by steliyan** [comment]
Any updates on this?
